packages/api/tsoa.json

```diff
@@ -5,7 +5,7 @@
 	},
 	"entryFile": "src/index.ts",
 	"noImplicitAdditionalProperties": "throw-on-extras",
-	"controllerPathGlobs": ["src/**/*Controller.ts"],
+	"controllerPathGlobs": ["src/**/*Controller.ts", "src/**/*InterfaceHttp.ts"],
```

Comment:
on veut garder le format en PascalCase ici ? plutôt que `.interface.http.ts` comme on fait pour les services ?

Reply:
Ba je sais pas je me suis dit que vu que les Controllers était tous en PascalCase on pouvais laisser comme ça !
Et aussi les services ce sont des instances de la class qui est exporter alors que la c'est la class non instanciée

Reply:
ok ça me va

Reply:
Historiquement c'est pour ça que certain fichier sont in PascalCase